numpy/core/numeric.py

```diff
@@ -3011,11 +3011,8 @@ class errstate(_errstate_baseclass):
     the context the error handling is set with `seterr` and `seterrcall`, and
     upon exiting it is reset to what it was before.
     
-    ..  versionchanged: 1.14.0
-        In Python 3, errstate is also usable as a function decorator, saving
-        a level of indentation if an entire function is wrapped.
-        
-        See :py:class:`contextlib.ContextDecorator` for more information.
+    ..  versionchanged: 1.14.0 
+        In Python 3, errstate is also usable as a function decorator (:py:class:`contextlib.ContextDecorator`)
```

Comment:
To confirm my hunch, it doesn't appear that we indent this description in any of the other uses in numpy - so perhaps keep the longer multi-paragraph description?